chaco/tools/legend_highlighter.py

```diff
@@ -1,13 +1,16 @@
-import operator
-
-import six
-import six.moves as sm
-
 # ETS imports
 from chaco.tools.api import LegendTool
 from traits.api import List, Float
 
 
+def _ensure_list(obj):
+    """ NOTE: The Legend stores plots in a dictionary with either single
+    renderers as values, or lists of renderers.
+    This function helps us assume we're always working with lists
+    """
+    return obj if isinstance(obj, list) else [obj]
```

Comment:
I briefly looked into the code for `Legend` to see if it made more sense to fix things there, but that did not appear to be the safe approach.